packages/kbn-alerts-ui-shared/src/alert_filter_controls/utils.ts

```diff
@@ -17,7 +17,7 @@ import { isEmpty, isEqual, pick } from 'lodash';
 import type { FilterControlConfig } from './types';
 
 export const getPanelsInOrderFromControlsState = (controlState: ControlGroupRuntimeState) => {
-  const panels = controlState.initialChildControlState;
+  const panels = controlState.initialChildControlState ?? {};
```

Comment:
It would be nice to have some tests for the util.

Reply:
+1